Source/JavaScriptCore/wasm/js/JSWebAssemblyArray.h

```diff
@@ -42,30 +42,28 @@ namespace JSC {
 class JSWebAssemblyArray final : public WebAssemblyGCObjectBase {
 public:
     using Base = WebAssemblyGCObjectBase;
-    static constexpr DestructionMode needsDestruction = NeedsDestruction;
-
-    static void destroy(JSCell*);
 
     template<typename CellType, SubspaceAccess mode>
     static CompleteSubspace* subspaceFor(VM& vm)
     {
         return vm.heap.webAssemblyArraySpace<mode>();
```

Comment:
Let's allocate it from cellSpace and remove `webAssemblyArraySpace`.

Reply:
I was going to do that in a follow up patch.